gulpfile.js

```diff
@@ -990,22 +986,35 @@ function concatFilesToString(files) {
  */
 function compileJs(srcDir, srcFilename, destDir, options) {
   options = options || {};
+
+  const entryPoint = path.join(srcDir, srcFilename);
+
+  // Transpile TS to Closure-annotated JS before actual bundling or compile.
+  if (options.typeScript) {
```

Comment:
Sad to see the auto-discovery of typescript components go away with this change and move to explicit options directive.

What's the reason?

Reply:
Makes sense. Happy to go this route and minimize differences.